index.html

```diff
@@ -2518,14 +2518,16 @@ <h3>
           When an <a>ImageResource</a> is used as an <dfn>icon</dfn>, a
           developer can hint that the image is intended to serve some special
           purpose in the context of the host OS (i.e., for better integration).
+          User agents SHOULD NOT use an icon other than for its stated purpose.
         </p>
-        <p>
-          For example, as a <a data-lt="badge purpose">badge</a> or "pinned"
-          icon that is visually distinct, in color or form, from an
-          application's launch icon. The user agent uses the value of the
-          <a>purpose</a> member as a hint to determine where and how an
-          <a>ImageResource</a> is displayed. Unless declared otherwise by the
-          developer, a user agent can use an icon for <a>any purpose</a>.
+        <p class="note">
+          For example, an icon with purpose <code>"badge"</code> may be used as
+          a <a data-lt="badge purpose">badge</a> or "pinned" icon that is
```

Comment:
```suggestion
          a <a data-lt="badge purpose">badge</a> or "<a>pinned</a>" icon that is
```

Reply:
Actually, I accepted this but I realised it doesn't make sense; "pinned" is not a concept in this doc, it's just being used as a word in this sentence that's quoted for emphasis. (I didn't change this sentence but I'll update it now so it doesn't look like a formal concept.)

Reply:
```suggestion
          a <a data-lt="badge purpose">badge</a> or "pinned" icon that is
```

Reply:
Hopefully the above lets it be reverted back. Sorry about that.